pep-0570.rst

```diff
@@ -385,6 +385,305 @@ as a rule is not LL(1). This is the reason the rule needs to be included in
 the existing typedargslist and varargslist (in the same way keyword-only
 arguments were introduced).
 
+=================
+How to teach this
+=================
+
+This PEP recommends adding a new subsection to the Python documentation,
+in the section "More on Defining Functions", where the rest of the
```

Comment:
Can we add a link to this section?

Looks like it refers to: https://docs.python.org/3/tutorial/controlflow.html#more-on-defining-functions